test/test_qi_metrics.py

```diff
@@ -32,3 +36,11 @@ def test_rdm2():
     h2_rdm2_dmrg = get_two_orbital_rdm(h2_mps, [1, 2])
 
     assert np.allclose(h2_rdm2, h2_rdm2_dmrg, atol=0.01)
+
+
+def test_get_all_mutual_information(water_DMRG):
+    water_mi = get_all_mutual_information(water_DMRG)
```

Comment:
This takes way too long for a unit test. Can this be replaced with a much smaller system?

Reply:
How long do you want it to take?